src/types/Filter.ts

```diff
@@ -0,0 +1,5 @@
+export enum Filter {
+  ALL = 'All',
+  ACTIVE = 'Active',
+  COMPLETED = 'Completed',
+}
```

Comment:
usually, enum items are not uppercase
```suggestion
export enum FilterBy {
  All = 'All',
  Active = 'Active',
  Completed = 'Completed',
}
```